feature-libs/user/profile/assets/translations/translations.ts

```diff
@@ -0,0 +1,10 @@
+import { TranslationChunksConfig, TranslationResources } from '@spartacus/core';
+import { en } from './en/index';
+
+export const userProfileTranslations: TranslationResources = {
+  en,
+};
+
+export const userTranslationChunksConfig: TranslationChunksConfig = {
+  user: ['updateEmailForm', 'register', 'loginForm', 'forgottenPassword'],
```

Comment:
```suggestion
  user: ['updateEmailForm', 'register', 'forgottenPassword'],
```

I think this should not be here

Reply:
👍